frontend/src/app/shared/components/file-links/file-link-list-item/file-link-list-item.html

```diff
@@ -14,20 +14,25 @@
   >
     <a
       class="spot-list--item-action spot-list--item-action_link op-file-list--item-action"
-      [ngClass]="{ 'disabled': disabled, 'view-not-allowed': !viewAllowed }"
+      [ngClass]="{
+        'disabled': disabled,
```

Comment:
do we need this if we have the one below? Same goes for `view-not-allowed`, I'd like ot see us move away from non-BEM classes.

Reply:
Yes, I struggled, if I should rework that now.

both are CSS overwrites for the file link list, which are from june and were introduced as non-BEM classes due to lack of knowledge on my part. ^^

You think, I should use this PR to change them?

Reply:
If it's not too much work, I'd just pop this into this PR. But if you feel like postponing this to the next, go ahead :)

Reply:
@b12f I removed the `disabled` class, which was no longer needed and refactored the `view-not-allowed` to a BEM-style modifier.